homeassistant/components/device_tracker/unifi.py

```diff
@@ -90,6 +97,11 @@ def _update(self):
             _LOGGER.error("Failed to scan clients: %s", ex)
             clients = []
 
+        # Filter clients to provided SSID list
+        if self._ssid_filter:
+            clients = filter(lambda x: x['essid'] in self._ssid_filter,
```

Comment:
List comprehension is preferred over `filter`. It's more verbose.

Reply:
Sure thing, I'm happy to update my PRs comments before a merge.

I can also open a new one with this change.

Reply:
This PR has already been merged so we can't continue here. You're welcome to open a new PR with that change.

Reply:
Opened: https://github.com/home-assistant/home-assistant/pull/12344